.github/workflows/release.yml

```diff
@@ -0,0 +1,164 @@
+name: Release new version of Optable SDK and demos
+on:
+  push:
+    tags:
+      - "v.*"
+
+jobs:
+  tests-prettier:
+    uses: ./.github/workflows/reusable-lint-test.yml
+
+  build:
+    needs: [tests-prettier]
+    uses: ./.github/workflows/reusable-build.yml
+
+  deploy-sdk-to-npm:
+    needs: [build]
+    runs-on: ubuntu-latest
+    env:
+      gcp-project-id: 118585658141
+      service-account: gh-ci-optable-web-sdk
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Download web artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: dist-web
+          path: browser/dist
+
+      - name: Download lib artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: dist-web
+          path: lib/dist
+
+        # Step will fail if the version is invalid, github.ref_name is the tag name (v.*.*.*)
+      - name: Patch version
+        run: ./scripts/patch-version.sh "${{ github.ref_name }}"
+
+      - name: Setup registry access
+        run: |
+          echo '@optable:registry=https://registry.npmjs.org/' > ~/.npmrc
+          echo "//registry.npmjs.org/:_authToken=${{ secrets.GITHUB_TOKEN }}" >> ~/.npmrc
+
+      - name: Publish to NPM
+        run: npm publish --access public
+
+  define-gcs-versions-to-update:
+    needs: [deploy-sdk-to-npm]
+    runs-on: ubuntu-latest
```

Comment:
same here: We should use the 2-core-ubuntu since it is included in our plan. I believe those are small jobs that won't need a more powerful one.